thoth/metrics_exporter/configuration.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# thoth-metrics-exporter
+# Copyright(C) 2020 Francesco Murdaca
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Configuration of metrics-exporter."""
+
+import logging
+import os
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Configuration:
+    """Configuration of metrics-exporter."""
+
+    # Prometheus
+    _URL = os.environ["PROMETHEUS_HOST_URL"]
+    _PROMETHEUS_SERVICE_ACCOUNT_TOKEN = os.environ["PROMETHEUS_SERVICE_ACCOUNT_TOKEN"]
+    _HEADERS = {"Authorization": f"bearer {_PROMETHEUS_SERVICE_ACCOUNT_TOKEN}"}
+    _PROM = PrometheusConnect(url=_URL, disable_ssl=True, headers=_HEADERS)
+
+    # Workflows backend namespace
+    _WORKFLOW_CONTROLLER_INSTANCE_BACKEND_NAMESPACE = os.environ["WORKFLOW_METRICS_BACKEND_PROMETHEUS_INSTANCE"]
+    _THOTH_BACKEND_NAMESPACE = os.environ["THOTH_BACKEND_NAMESPACE"]
+
+    # Workflows middletier namespace
+    _WORKFLOW_CONTROLLER_INSTANCE_MIDDLETIER_NAMESPACE = os.environ["WORKFLOW_METRICS_MIDDLETIER_PROMETHEUS_INSTANCE"]
+    _THOTH_MIDDLETIER_NAMESPACE = os.environ["THOTH_MIDDLETIER_NAMESPACE"]
+
+    # Workflows amun-inspection namespace
+    _WORKFLOW_CONTROLLER_INSTANCE_AMUN_INSPECTION_NAMESPACE = os.environ[
+        "WORKFLOW_METRICS_AMUN_INSPECTION_PROMETHEUS_INSTANCE"
+    ]
+    _THOTH_AMUN_INSPECTION_NAMESPACE = os.environ["THOTH_AMUN_INSPECTION_NAMESPACE"]
```

Comment:
These do not need to be private, they are used outside of the class.

Reply:
Updated :) Thanks!!

Reply:
The README file deserves some changes as well :)

Reply:
Done :D Thanks :)